server/surveyRdb/surveyRdbManager.js

```diff
@@ -37,6 +40,34 @@ const countTable = async (surveyId, tableName, filter, client = db) =>
 const queryRootTableByRecordKeys = async (survey, recordUuid, client = db) =>
   await TableViewQuery.queryRootTableByRecordKeys(survey, recordUuid, client)
 
+const exportTableToCSV = async (surveyId, tableName, cols, filter, output) => {
+  const csvStream = fastcsv.createWriteStream({ headers: true })
+  csvStream.pipe(output)
+
+  // 1. write headers
+  csvStream.write(cols)
+
+  let offset = 0
+  const limit = 10
```

Comment:
Limit at least 100 for csv export?

Reply:
sure, the limit to 10 is simply a mistake... we can limit the query even to 500 rows, it won't be too slow and it won't occupy too much memory